src/index.js

```diff
@@ -725,7 +729,13 @@ function addMutationObserver(settings) {
         }
     });
 
-    cssVarsObserver.observe(document.documentElement, {
+    let watchTarget = null;
+    if (settings.watchTarget) {
+        watchTarget = document.querySelector(settings.watchTarget);
```

Comment:
In the event that `watchTarget` is `null`, would it make sense to `console.warn` that the desired `watchTarget` failed to be selected?